README.md

```diff
@@ -88,6 +88,40 @@ reader.onload = function() {
 
 The uplifted library API is still pending... As of right now, a lot of the functions and properties are pretty easily found if you know your way around an OpenType font already by looking at the source as well as the tests, but that's not ideal - API docs [will be forthcoming](https://github.com/Pomax/Font.js/issues/93) but can always use help.
 
+That said, this section will keep getting expanded as the API gets consolidated.
```

Comment:
Nowhere near complete, but starting the API docs now means there's something to build out.